src/employees/utils/getEmployeesList.ts

```diff
@@ -87,3 +87,77 @@ function massageEmployee(employee: ApiEmployee) {
     officeName: employee.office_name,
   };
 }
+
+const offices = ['Trondheim', 'Oslo', 'Bergen'];
+const givenName = [
+  'Nora',
+  'Emma',
+  'Sofie',
+  'Olivia',
+  'Ella',
+  'Sofia',
+  'Maya',
+  'Lea',
+  'Frida',
+  'Ingrid',
+  'Noah',
+  'Oskar',
+  'Oliver',
+  'Lukas',
+  'Isak',
+  'Axel',
+  'Emil',
+  'Fillip',
+  'Jakob',
+  'Will',
+];
+const sureName = [
+  'Hansen',
+  'Johansen',
+  'Olsen',
+  'Larsen',
+  'Andersen',
+  'Pedersen',
+  'Nilsen',
+  'Kristiansen',
+  'Jensen',
+  'Karlsen',
+  'Johnsen',
+  'Pettersen',
+  'Eriksen',
+  'Berg',
+  'Haugen',
+  'Johannessen',
+];
```

Comment:
Dette var **veldig** norske navn, og muligens ikke gjenspeiler Variant helt. Kanskje disse skulle ha vært mer tull/random navn, litt som robohashen? 🤖

Reply:
Kan godt være hardkodet altså, bare tenkte at SSB topp 20 av robotnavn i stedet 😂 Eller noe annet.

Reply:
@123Inge @TrymVei Dere kan jo også ha bare en liste med masse random navn (søke etter "robot name generator"), så bare kombinere to av de for fornavn og etternavn:

Terra
Clank
Hammer
Crowby
osx
okutron

etc etc

Reply:
Yes, vi skal få noen bedre navn for bots

Reply:
Skal være mer robot aktige navn nå